components/graph/graph.js

```diff
@@ -127,10 +146,7 @@ GraphViewModel.prototype.loadNodesFromApi = function(callback) {
 
       self.edges(edges);
       self.nodes(nodes);
-
-      if (nodes.length > 0) {
```

Comment:
Wrapping with `Math.max` does not prevent it from beeing called.

Try the following in a console:
```js
var nodes = [];
Math.max(nodes[nodes.length - 1].cy(), 0);
```